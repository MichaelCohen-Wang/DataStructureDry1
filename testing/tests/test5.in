add_pirate 410 508 13
update_pirate_treasure 295 11
add_pirate 228 311 17
add_ship 272 11
remove_ship 433
add_ship 97 19
get_cannons 383
add_pirate 190 97 20