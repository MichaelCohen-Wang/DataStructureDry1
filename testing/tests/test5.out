add_pirate: FAILURE
update_pirate_treasure: FAILURE
add_pirate: FAILURE
add_ship: SUCCESS
remove_ship: FAILURE
add_ship: SUCCESS
get_cannons: FAILURE
add_pirate: SUCCESS
